pkg/plugin/plugin.go

```diff
@@ -285,51 +284,11 @@ func (e *AutoscaleEnforcer) Name() string {
 //
 // This function returns nil, nil if the pod is not associated with a NeonVM virtual machine.
 func (e *AutoscaleEnforcer) getVmInfo(logger *zap.Logger, pod *corev1.Pod, action string) (*api.VmInfo, error) {
-	vmName := util.TryPodOwnerVirtualMachine(pod)
-	if vmName == nil {
-		return nil, nil
-	}
```

Comment:
Do we still need this check here?

Asking because of the comment above the function:

> This function returns nil, nil if the pod is not associated with a NeonVM virtual machine.

Reply:
brought it back.